src/models/thelio-b4/repairs.md

```diff
@@ -0,0 +1,455 @@
+- [Replacing the bottom case fan](#replacing-the-bottom-case-fan)
+- [Replacing the GPU](#replacing-the-gpu)
+- [Removing the CPU shroud](#removing-the-cpu-shroud)
+- [Replacing the RAM](#replacing-the-ram)
+- [Replacing the M.2 drives](#replacing-the-m2-drives)
+- [Replacing the CPU fans](#replacing-the-cpu-fans)
+- [Replacing the CPU cooler/thermal paste and CPU](#replacing-the-cpu-cooler-and-cpu)
+- [Replacing the power supply](#replacing-the-power-supply)
+- [Replacing the Thelio-IO board](#replacing-the-thelio-io-board)
+- [Troubleshooting the power button](#troubleshooting-the-power-button)
+
+## Replacing the front accent strip:
+
+Thelio includes a customizable accent panel on the front of the case, which can be swapped to change the case's look and feel. The instructions for swapping the accent are also available in [video form](https://system76.com/r/diyaccentpanel).
+
+**Tools required:** None  
+**Time estimate:** 30 seconds  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to replace the front accent strip:
+
+1. Place the Thelio on the edge of the desk so the front side is hanging off of the desk.
+    - The Thelio can alternatively be lifted or tilted so the front of the computer is hovering above the desk.
+2. Slide the accent strip down to unlock it.
+    - The accent can be gripped at the bottom edge.
+3. Pull the accent strip off of the case, starting with the bottom edge.
+4. Place the new accent strip onto the front of the case and slide it up to lock it into place.
+
+![Front accent strip removal](./img/accent-strip-removal.webp)
+
+## Removing the top case:
+
+The top case can be removed to access the internal components.
+
+**Tools required:** Cross-head (Phillips) screwdriver (optional)  
+**Time estimate:** 1 minute  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to remove the top case:
+
+1. Remove the four outer screws holding the top case onto the machine.
+
+![Top case screws](./img/top-case-screws.webp)
+
+2. Slide the top case up and off of the machine.
+
+## Adding/removing 2.5" storage drives:
+
+Thelio B4 supports up to four 2.5" SATA III drives.
+
+**Tools required:** Cross-head (Phillips) screwdriver (optional)  
+**Time estimate:** 5 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to add/remove 2.5" storage drives:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case).
+2. Unscrew the two screws securing the drive bay's cover, highlighted green below.
+
+![2.5" drive cover and 2.5" drive screws](./img/25-drive-screws.webp)
+
+3. If you are adding a new drive, pop out a black plastic ring on the top crossbar and slide out four screws (per drive).
+4. Insert four screws into each 2.5" storage drive you wish to install.
+5. Slide each 2.5" drive into one of the slots leading to the Thelio-IO board.
+
+![2.5" drive installation](./img/25-drive-installation.webp)  
```

Comment:
It looks like this image is missing from the img directory. I'm only seeing the description text.

Reply:
I forgot that photo had a focus issue. I retook it today and will drop it in once I can get it off of my camera.